packages/core/src/components/menu/menuItem.tsx

```diff
@@ -41,9 +41,9 @@ export interface IMenuItemProps extends IActionProps, ILinkProps {
     submenu?: IMenuItemProps[];
 
     /**
-     * Width of "margin" from left or right edge of viewport. Submenus will
+     * Width of `margin` from left or right edge of viewport. Submenus will
      * flip to the other side if they come within this distance of that edge.
-     * This has no effect if omitted or if `useSmartPositioning={false}`.
+     * This has no effect if omitted or if `useSmartPositioning` is set to `false`.
```

Comment:
meh, this actually feels a little less readable to me, even though it is more of a sentence